pkgs/applications/misc/mupdf/default.nix

```diff
@@ -13,27 +14,43 @@ let
 
 
 in stdenv.mkDerivation rec {
-  version = "1.17.0";
+  version = "1.18.0";
   pname = "mupdf";
 
   src = fetchurl {
     url = "https://mupdf.com/downloads/archive/${pname}-${version}-source.tar.gz";
-    sha256 = "13nl9nrcx2awz9l83mlv2psi1lmn3hdnfwxvwgwiwbxlkjl3zqq0";
+    sha256 = "0rljl44y8p8hgaqializlyrgpij1wbnrzyp0ll5kcg7w05nylq48";
   };
 
   patches =
```

Comment:
Can you comment why you picked these patches? The upstream repo shows a few more patches that were added after the release. It isn't clear (from this discussion & the commit message) why these were picked.

Reply:
These seemed to be sensible post-release bugfixes that I deemed worth including. At the time of creating this PR, there were only a few patches.
We should not block this PR on the inclusion of further patches or dropping the patches already included, as there is already a RC for 1.18.1 which I prefer to put into a new PR.

Reply:
Ok, sounds reasonable enough.